<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>嵌入测试工具</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitterTabWidget">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"> QTabWidget::pane { /* The tab widget frame */
     border-top: 2px solid #C2C7CB;
 }

 QTabWidget::tab-bar {
     left: 5px; /* move to the right by 5px */
 }

 /* Style the tab using the tab sub-control. Note that
     it reads QTabBar _not_ QTabWidget */
 QTabBar::tab {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
     border: 2px solid #C4C4C3;
     border-bottom-color: #C2C7CB; /* same as the pane color */
     border-top-left-radius: 4px;
     border-top-right-radius: 4px;
     min-width: 8ex;
     padding: 2px;
 }

 QTabBar::tab:selected, QTabBar::tab:hover {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                 stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
 }

 QTabBar::tab:selected {
     border-color: #9B9B9B;
     border-bottom-color: #C2C7CB; /* same as pane color */
 }

 QTabBar::tab:!selected {
     margin-top: 2px; /* make non-selected tabs look smaller */
 }
</string>
       </property>
      </widget>
      <widget class="QFrame" name="frameConsole">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="rightMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBoxShowLog">
            <property name="text">
             <string>显示日志</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btnClearLog">
            <property name="toolTip">
             <string>清理日志</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/resource/imgs/broom_96px.png</normaloff>:/resource/imgs/broom_96px.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(131, 131, 131);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frameControl">
      <property name="styleSheet">
       <string notr="true">#frameControl{
background-color:white;
border-radius:12px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBoxEmbedCfg">
         <property name="styleSheet">
          <string notr="true">#groupBoxEmbedCfg{
	font: 12pt &quot;幼圆&quot;;
}</string>
         </property>
         <property name="title">
          <string>启动并嵌入配置</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="btnExistedCfg">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>选择已有配置</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>应用别名(可选)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditAppAlias"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>进程名称</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelProcessNameHint">
              <property name="styleSheet">
               <string notr="true">color: red;</string>
              </property>
              <property name="text">
               <string>(必需)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditProcessName"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>完整路径</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelProcessPathHint">
              <property name="styleSheet">
               <string notr="true">color: red;</string>
              </property>
              <property name="text">
               <string>(必需)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditProcessPath"/>
            </item>
            <item>
             <widget class="QToolButton" name="btnChoosePath">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>选择</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>启动等待时间</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelProcessNameHint_2">
              <property name="styleSheet">
               <string notr="true">color: red;</string>
              </property>
              <property name="text">
               <string>(必需)</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxWatingTime">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>秒(s)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>参数(可选)(多个参数空格分开)(带空格参数请加引号)：</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditProcessParam">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>窗口标题(可选)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditWndTitle"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>窗口类名(可选)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditClassName"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="rightMargin">
             <number>8</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxRestart">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>如果进程已经启动，杀掉进程后重启</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="labelHint">
            <property name="styleSheet">
             <string notr="true">color:red;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>138</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnStartNotEmbeded">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>启动(不嵌入)</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnStartAndEmbeded">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>启动 and 嵌入</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxOtherTests">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>GroupBox</string>
         </property>
         <widget class="QPushButton" name="btnGuogai">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>80</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>锅盖窗口测试</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>199</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="framebBottom">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelDateTime">
        <property name="styleSheet">
         <string notr="true">color: #CECECE;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1068</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelVersion">
        <property name="styleSheet">
         <string notr="true">color: #CECECE;</string>
        </property>
        <property name="text">
         <string>版本:v1.0.0.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labeAuthor">
        <property name="styleSheet">
         <string notr="true">color: #CECECE;</string>
        </property>
        <property name="text">
         <string>作者 yonghua.zhou@demxs.com</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
